feat: add scripts/install.sh

mikewuu opened the pull request:
closes #11 

- fixed to release 0.0.8. We should probably bump the version automatically, bundle the install script in a release? 

### to give it a go

```
curl 'https://raw.githubusercontent.com/seamapi/seam-cli/install-script/scripts/install.sh' | sudo sh
```

mikewuu committed:
feat: add scripts/install.sh

mikewuu committed:
fix missing curl_download

razor-x commented on scripts/install.sh:
```diff
@@ -0,0 +1,445 @@
+    if [ "$status_code" -eq 404 ]; then
+      log ""
+      log "Please report this issue:"
+      log "https://github.com/DopplerHQ/cli/issues/new?template=bug_report.md&title=[BUG]%20Unexpected%20404%20using%20CLI%20install%20script"
```
Why is doppler here?

mikewuu replied:
Because it was adapted from their script. Removed in https://github.com/seamapi/seam-cli/pull/65

razor-x commented:
@mikewuu If this was adapted from https://github.com/DopplerHQ/cli/blob/08cd5113f646ed98d4ca806c204428c68a69b8a1/scripts/install.sh then we need to adhere by their license: https://github.com/DopplerHQ/cli/blob/master/LICENSE

In particular, the requirements of section 4 need to be met. Since it's just one file, we can just put the license with the notice we changed the file with a link back to the source as a comment in the install script.

mikewuu commented:
> @mikewuu If this was adapted from https://github.com/DopplerHQ/cli/blob/08cd5113f646ed98d4ca806c204428c68a69b8a1/scripts/install.sh then we need to adhere by their license: https://github.com/DopplerHQ/cli/blob/master/LICENSE
> 
> In particular, the requirements of section 4 need to be met. Since it's just one file, we can just put the license with the notice we changed the file with a link back to the source as a comment in the install script.

Addressed in https://github.com/seamapi/seam-cli/pull/65
